cypress/component/DataSearch/dataset_search_table.spec.js

```diff
@@ -38,6 +42,56 @@ describe('Dataset Search Table tests', () => {
       cy.get('#header-checkbox').click();
       cy.contains('1 dataset selected from 1 study');
     });
+  });
+
+
+  describe('Data library filter by participant count tests', () => {
+
+    beforeEach(() => {
+      cy.initApplicationConfig();
+      cy.stub(TerraDataRepo, 'listSnapshotsByDatasetIds').returns({});
+      cy.clock();
+    });
+
+    function handler(request, searchText) {
+      if (JSON.stringify(request.body).includes(searchText)) {
+        request.reply(['filtered']);
+      } else {
+        request.reply([]);
+      }
+    }
+
+
+    it('When a participant count filter is applied the query is updated', () => {
+      cy.intercept(
+        {method: 'POST', url: '**/search/index'}, (req) => {
+          return handler(req, '{"range":{"participantCount":{"gte":null,"lte":50}}}');
+        }).as('searchIndex1');
```

Comment:
Is it required that `searchIndex1` must be different than `searchIndex2` in the other test? I would expect the alias names to be independent across tests so the same name could be used for both.

Reply:
good question, no I don't think so I just changed them to detangle as part of the debugging effort but I can change them back now